### PR TITLE
Master company config settings kme

### PR DESCRIPTION
**Pull request by kme-odoo**
Task : https://www.odoo.com/web?#id=27812&view_type=form&model=project.task&action=327&menu_id=4720
Pad : https://pad.odoo.com/p/r.ea3a82ad157ba392f03749b25c4e19ad

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:
## 

I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/sale/views/sale_config_settings_views.xml [hunk 1]
```diff
@@ -60,6 +60,11 @@
                             <field name="sale_note" class="oe_inline"/>
                         </div>
                     </group>
+                    <xpath expr="//div[last()]" position="after">
+                        <p class="text-muted" groups="base.group_multi_company">
+                            * These fields are company-dependant, you are setting the config for the YourCompany only.
```
Comment (Yenthe666) [inline addons/sale/views/sale_config_settings_views.xml]: @kme-odoo can you rephrase this please? the last part is confusing.

File: openerp/addons/base/res/res_company_view.xml [hunk 2]
```diff
@@ -50,7 +50,7 @@
                         </page>
                         <page name="configuration" string="Configuration">
                             <group>
-                                <group name="account_grp" string="Accounting"/>
```
Comment (tivisse) [inline openerp/addons/base/res/res_company_view.xml]: account_group is still used in xpath in account_sepa and l10n_us_check_printing

Comment (atp-odoo) [inline openerp/addons/base/res/res_company_view.xml]: we knew that. that's why we already replated see enterprise branch  :) 

https://github.com/odoo/enterprise/pull/489/files#diff-b785a5b9a1502707682065378001071eR8

Comment (tivisse) [inline openerp/addons/base/res/res_company_view.xml]: I just took the 2 branches and it still visible in :

/account_sepa/views/res_company_view.xml
/l10n_us_check_printing/views/res_company_view.xml

I'll check it.

File: addons/account/views/res_config_view.xml [hunk 3]
```diff
@@ -98,7 +98,32 @@
                             </div>
                         </div>
                     </group>
-                    <group string="Accounting &amp; Finance">
+                    <group string="Accounting &amp; Finance" name="accounting">
+                        <label for="use_anglo_saxon"/>
+                        <div>
+                            <field name="use_anglo_saxon" class="oe_inline"/>
+                            <span title='This field is company-dependant, you are setting the config for this company only.'><i class='fa fa-building-o' groups="base.group_multi_company"/></span>
```
Comment (tivisse) [inline addons/account/views/res_config_view.xml]: This is very strange to replicate this ugly icon and the same text so many times...

File: addons/account/views/company_view.xml [hunk 4]
```diff
@@ -6,13 +6,6 @@
...
             <field name="arch" type="xml">
-                <xpath expr="//group[@name='account_grp']" position="inside">
-                    <field name="anglo_saxon_accounting"/>
-                    <field name="bank_account_code_prefix" groups="base.group_no_one"/>
-                    <field name="cash_account_code_prefix" groups="base.group_no_one"/>
-                    <field name="accounts_code_digits" groups="base.group_no_one"/>
-                    <field name="tax_calculation_rounding_method" groups="base.group_no_one"/>
-                </xpath>
                 <notebook position="inside">
                     <page string="Overdue Payments">
```
Comment (tivisse) [inline addons/account/views/company_view.xml]: This should be moved too

File: addons/project/models/res_config.py [hunk 5]
```diff
@@ -8,6 +8,9 @@ class ProjectConfiguration(models.TransientModel):
     _name = 'project.config.settings'
     _inherit = 'res.config.settings'
 
+    company_id = fields.Many2one('res.company', string='Company', required=True,
+        default=lambda self: self.env.user.company_id)
+    project_time_mode_id = fields.Many2one(related='company_id.project_time_mode_id', default=lambda self: self.env.user.company_id.project_time_mode_id)
```
Comment (tivisse) [inline addons/project/models/res_config.py]: Why moving into project ?
